lib/docx/body-reader.js

```diff
@@ -127,13 +127,19 @@ function BodyReader(options) {
             var properties = {
                 type: "paragraphProperties"
             };
-            
+
+            for (var i = 0; i < element.children.length; i++) {
+                if (element.children[i].name == "w:pageBreakBefore") {
+                    return readXmlElement(element.children[i]);
+                }
+            }
```

Comment:
The effect of this code would seem to be to ignore all other paragraph properties. Do paragraphs with `w:pageBreakBefore` not have any other content in? Otherwise, ignoring those properties seems like incorrect behaviour.